subsystems/PistonControl.java

```diff
@@ -0,0 +1,22 @@
+package org.frc5587.lib.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+
+import static edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class PistonControl extends SubsystemBase{
+    private DoubleSolenoid[] doubleSolenoidPistons;
+
+    public PistonControl(int[] pistonIDS) {
+        for(int i = 0; i < pistonIDS.length; i+=2) {
+            doubleSolenoidPistons[i] = new DoubleSolenoid(i, i+1);
+        }
+    }
+
+    public void pistonsSet(Value direction) {
```

Comment:
I like this! However, you should also include `extend()` and `retract()` methods that just call this method with the correct direction